package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "handle-sdk",
-  "version": "0.4.27",
+  "version": "0.4.28",
```

If only we had a CI to update this automatically :eyes: 
```suggestion
  "version": "0.4.29",
```